src/auslib/web/public/client.py

```diff
@@ -230,6 +230,8 @@ def get_update_blob(transaction, **url):
     response = make_response(xml)
     response.headers["Cache-Control"] = app.cacheControl
     response.headers.extend(get_aus_metadata_headers(eval_metadata))
+    if query["product"] == "GMP":
```

Comment:
Let's move this to a list of products we enable content signatures for in https://github.com/mozilla-releng/balrog/blob/main/uwsgi/public.wsgi (you can then find it in `app.config`).

Reply:
I like that idea - thanks. Wasn't sure if there was any reason to add the new symbol in admin.wsgi also, so left that out.

Reply:
Nope - the content signatures only apply to the public app, so no need to add it there.